server/app/views/admin/questions/QuestionsListView.java

```diff
@@ -187,22 +188,45 @@ private Pair<DivTag, ImmutableList<Modal>> renderAllQuestionRows(
           renderQuestionCard(card, activeAndDraftQuestions, request);
       if (isQuestionPendingDeletion(card, activeAndDraftQuestions)) {
         archivedQuestionRows.add(rowAndModals.getLeft());
+      } else if (getDisplayQuestion(card).isUniversal()
+          && settingsManifest.getUniversalQuestions(request)) {
+        universalQuestionRows.add(rowAndModals.getLeft());
       } else {
         nonArchivedQuestionRows.add(rowAndModals.getLeft());
       }
       modals.addAll(rowAndModals.getRight());
     }
 
-    DivTag questionContent =
-        div(
-            div()
-                .withClass(ReferenceClasses.SORTABLE_QUESTIONS_CONTAINER)
-                .with(nonArchivedQuestionRows.build()));
-    if (!archivedQuestionRows.build().isEmpty()) {
+    ImmutableList<DomContent> universalQuestionContent = universalQuestionRows.build();
+    ImmutableList<DomContent> nonArchivedQuestionContent = nonArchivedQuestionRows.build();
+    ImmutableList<DomContent> archivedQuestionContent = archivedQuestionRows.build();
+    DivTag questionContent = div();
+    if (!universalQuestionContent.isEmpty()) {
+      questionContent.with(
+          div()
+              .withId("questions-list-universal")
+              .withClasses(ReferenceClasses.SORTABLE_QUESTIONS_CONTAINER)
+              .with(h2("Universal questions").withClasses("mt-8", "font-semibold"))
```

Comment:
suggestion (non-blocking):  Since you reuse the "mt-8" and "font-semibold" classes on each of these h2s, it could be nice to add a style to our `BaseStyles` such as `SEMIBOLD_HEADER`.